question.html

```diff
@@ -98,63 +98,75 @@ <h1 style="font-family: 'Bubblegum Sans', cursive;"><b>Let's ASK!</b></h1>
 	</span>
 	</form>
 </div>
-
-<div class="container">
-	<div class="list-group" id="todo-list">
-		<!-- iterate questions -->
-		<span class="list-group-item" ng-repeat="todo in todos | filter:input | questionFilter:maxQuestion" ng-class="{completed: todo.completed, editing: todo == editedTodo}" style="background-color: #F5F5F5">
-			<!-- display questions -->
-			<h4 class="list-group-item-heading">
-				<span class="label label-danger" ng-show="todo.new">New</span>
-				<span class="badge">{{todo.like}}</span>
-				<span class="badge">{{-todo.dislike}}</span>
-				<span ng-class="{completed: todo.completed}">{{todo.head}}</span>
-			</h4>
-			<h5 class="list-group-item-text">{{todo.desc}}</span> </h5>
-			<small style="margin-top:5px; color:#848484"><em> created: {{todo.timestamp | date:'medium' }} <span ng-show="todo.tags">tags: {{todo.tags}}</span> replies: {{todo.replies}} </em></small>
-
-			<!-- display buttons for question -->
-			<div>
-				<button class="btn btn-default btn-xs glyphicon glyphicon-hand-up" ng-click="doLike(todo)" ng-disabled="$storage[todo.$id]">Like</button>
-				<button class="btn btn-default btn-xs glyphicon glyphicon-hand-down" ng-click="doDislike(todo)" ng-disabled="$storage[todo.$id]">Disike</button>
-				<button ng-show="isAdmin" class="btn btn-default btn-xs glyphicon glyphicon-check" ng-click="toggleCompleted(todo)">Toggle Completed</button>
-				&nbsp;
-				<button ng-show="isAdmin" class="btn btn-default btn-xs glyphicon glyphicon-trash" aria-hidden="true" ng-click="removeTodo(todo)">Del</button>
+	<div class="container">
+	<!-- Question Bounder -->
+	<div class="list-group" style="background-color: #FFFFFF; padding: 10px">
+		<!-- Iterate Questions -->
+		<div class="media" ng-repeat="todo in todos | filter:input | questionFilter:maxQuestion" ng-class="{completed: todo.completed, editing: todo == editedTodo}">
+			<!-- Display Avatar -->
+			<div class="media-left">
+				<a href="#">
+					<img class="media-object" src="http://www.tiptopglobe.com/forum/images/avatars/gallery/Simpsons/simpsons_santas_little_help.gif" alt="...">
+				</a>
```

Comment:
You have not commented on this. 
We can already add this avatar feature to our master branch, although this is the wrong moment to do so and we might need to remove it again at a later phase.